### ISSUE TITLE
sudo: PERM_ROOT: setresuid(0, -1, -1): Permission denied

### ISSUE THREAD

**Issue by dobrzann** [meefik/linuxdeploy#1379]
Can anyone help me with solving an issue related to using the sudo command? I've tried installing various systems on different smartphones using Linux Deploy, and on each one, I encounter an error when attempting to use sudo: 

sudo: PERM_ROOT: setresuid(0, -1, -1): Permission denied
sudo: error initializing audit plugin. 

I've tried editing the sudoers file, but it doesn't change anything. The only workaround I've found is either changing the username to root or using the pre-installed doas command, but this doesn't allow me to fully utilize tools like Kiauh for installing Klipper.

Please, anyone with knowledge that could help solve this problem, I'm seeking assistanc

**Comment by sagisar1** [comment]
Do you have an apache2 web server involved? if yes I probably know what is the problem.
Otherwise, try to do make a trace of the syscalls with the **_strace_** command to identify the root cause of the problem

**Comment by btlzr** [comment]
I'm encountering the same issue

**Comment by sagisar1** [comment]
After researching, i found that the reason for this error is the fact that the apache2 module mpm_itk is enabled. when disabling it, and restarting the server, i was able to do sudo su for example without any errors, although i had the same capabilities (from the output of capsh --print). 

try this if relevant @btlzr @dobrzann 
TL;DR - you need to disable mpm_itk apache2 module
